docs/security.html

```diff
@@ -199,6 +198,7 @@
                     <li>ssl.cipher.suites (Optional). A cipher suite is a named combination of authentication, encryption, MAC and key exchange algorithm used to negotiate the security settings for a network connection using TLS or SSL network protocol.</li>
                     <li>ssl.enabled.protocols=TLSv1.2,TLSv1.1,TLSv1. It should list at least one of the protocols configured on the broker side</li>
                     <li>ssl.truststore.type=JKS</li>
+                    <li>ssl.truststore.password=test1234</li>
```

Comment:
We could replace quite a bit of our own documentation if that's the approach we wanted to take. All the keytool commands are not really specific to Kafka. I think the goal is to provide great docs that help people get started and avoid common surprises.

When this was mandatory, people were likely to do the safer thing. With this PR, it becomes optional and people are less likely to do it and we can't provide a good default. If we don't at least provide them with a reason why they should do it. Thinking about it some more, maybe we should actually leave it where it was and add a comment saying that it's optional, but recommended unless you know what you're doing.